50SAT/SPEX$50SAT.tex

```diff
@@ -0,0 +1,353 @@
+%% *************************************************************************
+%%
+%% This is an RIT Space Exploration Standard defining guidelines for content
```

Comment:
Help text should probably be deleted before submitting for review. It makes reviewing much harder when you have to scroll through a ton of comment lines.